### ISSUE TITLE
Please update the file spec to version 0.99

### ISSUE THREAD

**Issue by skykapok** [ephtracy/voxel-model#13]
Magicavoxel 0.99 supports world editor and render settings saved in vox file. Please update the file spec.

**Comment by manspaniel** [comment]
Yes this would be really useful. Is the format currently being used in 0.99 final? I’ve partially deciphered how the new chunk types work and relate to each other, but would be great to know if it’s going to change much in the next update.
